twitter-as.js

```diff
@@ -1,3 +1,5 @@
+global.Promise = require('bluebird');
```

Comment:
Given than `Promise` is already built-in to the node runtime, it would be clearer to `require` it in each file it is used. The distinction is quite important and it would be useful to know during reviews like this, without going to look for `global.Promise`.